src/Serilog.Sinks.Loggly/Properties/AssemblyInfo.cs

```diff
@@ -2,7 +2,7 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.1.0.0")]
```

Comment:
The fact that we are changing the API (LogglySink's constructor) may require a major version change; Alternatively, a second constructor and a fallback ( this() ) call could maybe prevent that from occurring.